knapsack/unbounded_knapsack.py

```diff
@@ -0,0 +1,81 @@
+"""
+    Unbounded Knapsack Problem :
+    https://www.geeksforgeeks.org/unbounded-knapsack-repetition-items-allowed/
+
+    0-1 Knapsack Problem :
+    https://en.wikipedia.org/wiki/Knapsack_problem
+    https://www.geeksforgeeks.org/0-1-knapsack-problem-dp-10/
+
+    The unbounded knapsack problem is an extension of the 0-1 knapsack problem.
+    The main difference is that it is possible to utilize an infinite number of items
+    in the unbounded knapsack problem.
+    In other words, the problem statement is :
+
+    Given a knapsack of capacity W
+    and n types of items with certain values and weights,
+    calculate the maximum total value of items the knapsack can carry.
+    It is allowed to use unlimited number of instances of an item.
+"""
+
+
+def knapsack(capacity: int, items_weights: list[int], items_values: list[int]) -> int:
+    """
+    Returns the maximum value of items that can be put in a knapsack of a capacity c.
+    Each type of item has a specific weight and value.
+    It is possible to carry unlimited number of instances of an item.
+
+    >>> c = 70
+    >>> items_values = [40, 900, 120]
+    >>> items_weights = [10, 20, 30]
+    >>> knapsack(c, items_weights, items_values)
```

Comment:
```suggestion
    >>> knapsack(capacity=70, weights=(40, 900, 120), values=(10, 20, 30))
```
Please repeat for the other tests.

Please add tests for:
`capacity = 0`
`capacity - -5`
`len(values)` != `len(weights)`
`len(values) == 5` and `len(weights) == 0`
etc.